assets/performanceprofile/scripts/clear-irqbalance-banned-cpus.sh

```diff
@@ -11,15 +11,16 @@ NONE=0
 
 [ ! -f "${IRQBALANCE_CONF}" ] && exit 0
 
-${SED} -i '/^\s*IRQBALANCE_BANNED_CPUS\b/d' "${IRQBALANCE_CONF}" || exit 0
+${SED} -i '/^\s*IRQBALANCE_BANNED_CPULIST\b/d' "${IRQBALANCE_CONF}" || exit 0
 # CPU numbers which have their corresponding bits set to one in this mask
 # will not have any irq's assigned to them on rebalance.
 # so zero means all cpus are participating in load balancing.
```

Comment:
This comment needs to be adjusted to reflect the change to `IRQBALANCE_BANNED_CPULIST`.  For example "so zero means all cpus are participating in load balancing" is no longer true.